tendrl/gluster_integration/objects/definition/gluster.py

```diff
@@ -94,6 +94,15 @@
         integration_id:
           help: "Tendrl managed/generated cluster id for the sds being managed by Tendrl"
           type: String
+        integration_name:
```

Comment:
I rebased my branch on top of master . I feel Tendrl:spec-28-logging-coverage branch is not upto date with the master .